cogs/msg.py

```diff
@@ -74,7 +76,22 @@ async def on_message(self, message):
         if len(result) == 0:
             return
 
-        await message.channel.send(result[0][1])
+        text = result[0][1]
+
+        def isUrl(url):
+            try:
+                result = urlparse(url)
+                return all([result.scheme, result.netloc])
+            except:
+                return False
+
+        rest = message.clean_content[search.end():].strip()
+        if isUrl(text) and len(rest) > 0:
+            rest = re.sub(r'\s+', '_', rest)
+            rest = quote(rest, safe='/')
+            text = f"https://api.memegen.link/images/custom/_/{rest}.gif?background={text}"
```

Comment:
This looks prone to injecting pings and other fun things. However, I'm not sure how to sanitize it best.

Reply:
I added both an escape using discord.py's `escape_mentions` and also looked into better escaping for the memegen API